src/Umbraco.Web.UI.Client/src/views/components/forms/umb-checkbox.html

```diff
@@ -0,0 +1,15 @@
+<label class="checkbox umb-form-check umb-form-check--checkbox">
+    <input type="checkbox" name="{{name}}"
```

Comment:
Maybe `name` attribute should also be prefixed `checkbox-` as radiobutton, so `checkbox-{{name}}`. Maybe also change `radiobuttons-{{name}}` to `radio-{{name}}` to keep it consistent?

Reply:
Love consistency. I thought that it would be fine just to remove the prefix, couldn't see the need for it. What do you think?

Reply:
@nielslyngsoe seems to be great. I wonder how it will in repeatable editors, e.g. a custom one or in Nested Content.

For example Nested Content with 3 items using a document type with 3 properties: Name (textstring), Category (checkbox list) and an Image (media picker).

If selecting at category in Nested Content item 2 og 3, does than mean the checkbox is checked in Item 1, because the value of the `name` attribute is the same? Or does the Items in nested content add a prefix/suffix to alias of the Nested Item?

Reply:
Furthermore could the ng-repeat in checkbox list and radiobutton list be consistent?

E.g. `<li ng-repeat="item in items track by item.id">`.
In checkboxlist it track by `key` and in radiobutton list it track by `id`.

https://github.com/umbraco/Umbraco-CMS/blob/31eefe1cf94a21ee1d509ce6dd399648c76be05d/src/Umbraco.Web.UI.Client/src/views/propertyeditors/checkboxlist/checkboxlist.html

https://github.com/umbraco/Umbraco-CMS/blob/31eefe1cf94a21ee1d509ce6dd399648c76be05d/src/Umbraco.Web.UI.Client/src/views/propertyeditors/radiobuttons/radiobuttons.html